README.md

```diff
@@ -55,7 +55,7 @@ There are multiple ways of installing.
 - GitHub (For Firefox & Firefox Forks): Download from the GitHub repository [here](https://github.com/STForScratch/ScratchTools/zipball/master). After the `.zip` file is downloaded, unpack it. Then, with the folder, go to `about:debugging`, click "This Firefox" and click "Load temporary extension", go into the extension folder and select the `manifest.json`.
 
 > [!WARNING]
-> Extensions loaded this way onto Safari indeed temporary. Once you close the window, it will be gone. In addition, ScratchTools is still outdated on Firefox at the time of writing this.
+> Extensions loaded this way onto Safari are temporary. Once you close the window, it will be gone.
```

Comment:
I feel like this could still be explained a bit better but I'm still not completely sure how extension installation on Safari works

Reply:
I feel like the changes you made fix it up pretty well.